test/scale/azdisk_scale_test.go

```diff
@@ -55,7 +55,7 @@ func scaleTests(isMultiZone bool) {
 		testDriver = driver.InitAzureDiskDriver()
 	})
 
-	ginkgo.It("Scale test scheduling and starting multiple pods with a persistent volume.", func() {
+	ginkgo.It("Scale test scheduling and starting multiple pods with persistent volume(s).", func() {
```

Comment:
Should we also update the scale test name in Makefile: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/main_v2/Makefile#L354 ?

Reply:
Yes, thanks for pointing out.